pkg/transformers/literal_string_transformer.go

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package transformers
+
+import (
+	"errors"
+	"fmt"
+)
+
+type LiteralStringTransformer struct {
+	literal string
+}
+
+var (
+	literalStringTransformerParams = []string{"literal"}
+	errLiteralStringCannotBeEmpty  = errors.New("literal_string_transformer: literal parameter cannot be empty")
+)
+
+func NewLiteralStringTransformer(params Parameters) (*LiteralStringTransformer, error) {
+	if err := ValidateParameters(params, literalStringTransformerParams); err != nil {
+		return nil, err
+	}
+
+	literal, _, err := FindParameter[string](params, "literal")
+	if err != nil {
+		return nil, fmt.Errorf("literal_string_transformer: literal must be a string: %w", err)
+	}
+	if literal == "" {
+		return nil, errLiteralStringCannotBeEmpty
```

Comment:
maybe we should allow empty strings, but keep disallowing not providing a string literal?

Reply:
Hmm, yeah, not sure if this should be an error, an empty value could be the desired behaviour.